modules/features/people_content_type/people_content_type_listing.inc

```diff
@@ -169,7 +169,7 @@ function _people_content_type_listing($view_mode, $type, $department, $filter_1,
         // prepare contact section
         $contact = '';
         if (!empty($output['nodes'][$nid]['field_person_email'])) {
-          $contact .= '<div class="table-person-email"><i class="fa fa-envelope"></i> <a href="mailto:' . render($output['nodes'][$nid]['field_person_email']) . '">Email</a></div>';
+          $contact .= '<div class="table-person-email"><i class="fa fa-envelope"></i> <a href="mailto:' . trim(render($output['nodes'][$nid]['field_person_email'])) . '" title="' . trim(render($output['nodes'][$nid]['field_person_email'])) . '">Email</a></div>';
```

Comment:
I'm pretty sure the idea here was to replace the text of the link as well as the href and title.